src/components/UploadBar/style.scss

```diff
@@ -105,6 +105,8 @@ $file-upload-progress: "file-upload-progress";
       padding-top: 16px;
       width: 100%;
       max-height: inherit;
+      display: flex;
+      flex-direction: column;
```

Comment:
`flex: 0 1 auto;` ?

Reply:
it was tested and seems that there is no need for it
I'm totally against adding not needed code